app/utils/custom-fields.ts

```diff
@@ -183,9 +206,17 @@ export const getCustomFieldDisplayValue = (
   hints?: ClientHint
 ): string => {
   if (value.valueDate) {
-    return hints
-      ? getDateTimeFormatFromHints(hints).format(new Date(value.valueDate))
-      : format(new Date(value.valueDate), "PPP"); // Fallback to default date format
+    if (hints) {
+      const dateFormatter = getDateTimeFormatFromHints(hints, {
+        year: "numeric",
+        month: "2-digit",
+        day: "2-digit",
+      }).format(new Date(value.valueDate));
+      const [month, day, year] = dateFormatter.split("/");
+      // Rearrange the components into YYYY-MM-DD format
+      return `${year}-${month.padStart(2, "0")}-${day.padStart(2, "0")}`;
```

Comment:
We don't do that. The idea is that the formatting is done based on the user locale and timezone. That is what our dateformatters do. You are formatting the date and then hardcoding a different format that doesnt fit with our app.

Here are some examples. My PC has dates in american style:

![image](https://github.com/user-attachments/assets/b12e9837-485a-4fc2-9ce5-922af9c13ced)
![image](https://github.com/user-attachments/assets/b4c8ded6-d7f7-48e4-9534-9df226aa6225)
![image](https://github.com/user-attachments/assets/f699f356-3f40-41dd-a8af-ab01b62d40f1)

As you can see they follow the same format everywhere, except the 1 place where we have the custom field where you are hardcoding the value.